composure.sh

```diff
@@ -275,7 +275,11 @@ cite ()
 
   typeset keyword
   for keyword in "$@"; do
-    eval "$keyword() { :; }"
+    if shopt -q expand_aliases; then
```

Comment:
The `shopt` builtin is bash-only, and is [not supported for zsh](https://github.com/ohmyzsh/ohmyzsh/issues/8614#issuecomment-584708629).